src/PhpWord/TemplateProcessor.php

```diff
@@ -1085,7 +1085,7 @@ protected function indexClonedVariables($count, $xmlBlock)
     {
         $results = array();
         for ($i = 1; $i <= $count; $i++) {
-            $results[] = preg_replace('/\$\{(.*?)\}/', '\${\\1#' . $i . '}', $xmlBlock);
+            $results[] = preg_replace('/\$\{([^:]*?)(:.*?)?\}/', '\${\1#' . $i . '\2}', $xmlBlock);
```

Comment:
I was just wondering why you're only using one `\` compared to the old regex and to my changes in https://github.com/PHPOffice/PHPWord/pull/1829:

`$results[] = preg_replace('/\$\{(.*?)(:([^}])*)?\}/', '\${\\1#' . $i . '\\2}', $xmlBlock);`

I'm not that much into this regex at the moment, so I don't know how they differ.

btw, this pr will also fix https://github.com/PHPOffice/PHPWord/issues/1624

Reply:
@Amerlander that's because in a single quoted string you don't have to double the \
https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.single

Reply:
Thank you for explanation and for the merge! :)